### ISSUE TITLE
Add book from Open Library

### ISSUE THREAD

**Issue by shawn1thompson** [fenimore/badreads#15]
After searching for a title in Open Library and successfully finding the book, I click on the search suggestion and am taken back to the home screen. I'm unable to add it.

**Comment by fenimore** [comment]
Oh that is strange behavior! Thank you for reporting it. Do you have an example search i can use to reproduce this? And i figure you're on the latest version

**Comment by shawn1thompson** [comment]
Yes, a [book](https://openlibrary.org/books/OL32982398M/Servant_of_the_Loving_One) I recently added. And, yes, I am; just found your app and am looking forward to using it. Thanks.

**Comment by fenimore** [comment]
Perfect thank you! I think i found the issue, the app crashed because of the ISBN, but i think it should be working for this book now. In the future it shouldn't crash though because of this, I'll have to make it more resilient. Thanks again! I'll let this thread know when the app is totally fixed
